frontend/src/pages/projects/screens/detail/ProjectDetailsSidebar.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {
+  JumpLinks,
+  JumpLinksItem,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+} from '@patternfly/react-core';
+import { ProjectSectionID } from './types';
+import { ProjectSectionTitles } from './const';
+
+const ProjectDetailsSidebar: React.FC = ({ children }) => {
+  const jumpLinkItems = [
+    { id: ProjectSectionID.WORKSPACE, title: ProjectSectionTitles[ProjectSectionID.WORKSPACE] },
+    { id: ProjectSectionID.STORAGE, title: ProjectSectionTitles[ProjectSectionID.STORAGE] },
+    {
+      id: ProjectSectionID.DATA_CONNECTIONS,
+      title: ProjectSectionTitles[ProjectSectionID.DATA_CONNECTIONS],
+    },
+  ];
+
+  return (
+    <Sidebar hasGutter>
+      <SidebarPanel variant="sticky">
+        <JumpLinks isVertical label="Jump to section" scrollableSelector="#project-details-list">
+          {jumpLinkItems.map(({ id, title }) => (
+            <JumpLinksItem key={`link-${id}`} href={`#${id}`}>
+              {title}
+            </JumpLinksItem>
+          ))}
```

Comment:
Please avoid creating extra items you can extrapolate from anyways. Data is best done when there is only a single source of truth. Untyped Arrays & Object creation are also very bad (`jumpLinkItems`), avoid those as much as you can. Maybe we can do something with TS to make it yell at untyped items.

Suggested change:
```
{Object.values(ProjectSectionID).map((section) => (
  <JumpLinksItem key={section} href={`#${section}`}>
    {ProjectSectionTitles[section]}
  </JumpLinksItem>
)}
```

Reply:
I did try this method but I used it in a wrong way at that time and rendered nothing 😞 now it works, thanks